### PR TITLE
#48 fix app broken when duplicate emails in the mailto parameter

### PR DESCRIPTION
**Pull request by rezk2ll**
image

resolves https://github.com/OpenPaaS-Suite/esn-frontend-mailto/issues/48

### CODE REVIEW

**Review by MichaelBailly — APPROVED** [review]

File: src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.js [hunk 1]
```diff
@@ -262,4 +263,10 @@ angular.module('linagora.esn.unifiedinbox')
         emailSendingService.removeReadReceiptRequest(self.message);
       }
     }
+
+    function _RemoveDuplicateTo(email) {
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.js:267]:
```suggestion
    function _removeDuplicateRecipients(message) {
```

File: src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.js [hunk 2]
```diff
@@ -50,6 +50,7 @@ angular.module('linagora.esn.unifiedinbox')
     /////
 
     function $onInit() {
+      _RemoveDuplicateTo(self.message);
```
Comment (tuanlc) [inline src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.js:53]: Write tests for this

**Review by tuanlc — DISMISSED** [review]
Please refactor your commit message to have the correct ticket link that is clickable to redirect to the ticket.

**Review by MichaelBailly — DISMISSED** [review]

**Review by piavgh — COMMENTED** [review]
The commit message should be `OpenPaaS-Suite/esn-frontend-mailto#48...`, not `OpenPaaS-Suite:esn-frontend-mailto#48...`. 
`/`, not `:`

File: src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.spec.js [hunk 3]
```diff
@@ -186,6 +186,20 @@ describe('The inboxComposerController controller', function() {
       expect(ctrl.onTitleUpdate).to.have.been.calledWith({ $title: 'subject' });
     });
 
+    it('should remove duplicate email adresses in the To property', () => {
```
Comment (chamerling) [inline src/linagora.esn.unifiedinbox/app/components/composer/composer.controller.spec.js:189]: Should also have a test when there are multiple email values, not only the same as this one

**Review by chamerling — CHANGES_REQUESTED** [review]

**Comment by rezk2ll** [comment]
> The commit message should be `OpenPaaS-Suite/esn-frontend-mailto#48...`, not `OpenPaaS-Suite:esn-frontend-mailto#48...`.
> `/`, not `:`

done ty

**Review by huy-ta — DISMISSED** [review]

**Review by piavgh — CHANGES_REQUESTED** [review]
The 2nd commit message is incorrect. It references another issue

**Review by tuanlc — APPROVED** [review]

**Review by piavgh — APPROVED** [review]

**Pull request merged**